mahjong/hand_calculating/tests/tests_scores_calculation.py

```diff
@@ -241,3 +280,15 @@ def test_kiriage_mangan(self):
 
         result = hand.calculate_scores(han=3, fu=60, config=config)
         self.assertEqual(result['main'], 12000)
+
+
+if __name__ == '__main__':
```

Comment:
I think this one is not needed here. You can run tests from root folder with `python -m unittest discover .`